src/snowflake/snowpark/_internal/packaging_utils.py

```diff
@@ -406,6 +430,9 @@ def invert_downloaded_package_to_entry_map(
                 ):  # Implies the relative_path is a file name at the base directory
                     record_entry = relative_path
 
+                if "\\" in record_entry:
+                    record_entry = record_entry.replace("\\", "/")
+
```

Comment:
Fix for Windows issue: Files returned by glob.glob have "\\" in the paths returned while the RECORD file follows the forward slash path convention.

Reply:
This is no longer covered by tests because we do not run tests on Windows